hydroshare/local_settings.py

```diff
@@ -76,7 +76,7 @@
 
 
 # CartoCSS
-CARTO_HOME='/home/docker/node_modules/carto'
+CARTO_HOME='/node_modules/carto'
```

Comment:
@mjstealey This CARTO_HOME is only used by ga_resources, so this may not matter at all. But since hydro-service account's home is set to be /hydroshare, it is probably better to set this CARTO_HOME to /hydroshare/node_modules/carto to correspond to the initial setting for /home/docker?